sdk/monitor/azquery/models.go

```diff
@@ -75,12 +77,12 @@ type Body struct {
 	// REQUIRED; The query to execute.
 	Query *string `json:"query,omitempty"`
 
+	// A list of workspaces to query in addition to the primary workspace.
+	AdditionalWorkspaces []*string `json:"workspaces,omitempty"`
```

Comment:
We should make it clear what the acceptable values are here. As I recall, workspace IDs are just one option. Read here for more info: https://github.com/Azure/azure-sdk-for-python/tree/main/sdk/monitor/azure-monitor-query#query-multiple-workspaces

Reply:
I am pretty sure it only accepts workspace IDs. I tried testing with a resource URI and got a BadArgumentError: "The given resource id for microsoft.appconfiguration/configurationstores is not for a Log Analytics resource"